applications/gitea/README.md

```diff
@@ -0,0 +1,159 @@
+
+```bash
+replicated cluster create \
+    --distribution embedded-cluster \
+    --instance-type r1.xlarge \
+    --disk 100 \
+    --license-id xxxxxx \
+    --ttl 4h \
+    --name postgres-cluster
+
+## Verify cluster creation
+replicated cluster ls
+
+## Access cluster shell
+replicated cluster shell <cluster-id>
+
+## Port forward the kotsadm to install the storagebox helm chart with PostgresSQL enabled
+kubectl port-forward svc/kotsadm 3000:3000 -n kotsadm
+```
+
+Login to the kotsadm dashboard with default password `password` and configure the storagebox helm chart with PostgresSQL only enabled. After the helm chart is installed, the PostgresSQL database is ready to be used by Gitea.
+
+#### 2. Gitea Cluster
+Create a new cluster using Replicated Compatibility Matrix with no pre-installed Kubernetes. The cluster will be used to deploy Gitea Helm chart.
+
+```bash
+## first, find the network id of the PostgresSQL cluster
+replicated network ls
+---
+ID          NAME                           STATUS          CREATED                           EXPIRES                           OUTBOUND
+709a523d    postgres-cluster                running         2025-02-18 17:06              2025-02-18 21:12              -    
+---
+```
+
+Use the network ID to create a new cluster with the following command:
+```bash
+replicated vm create --distribution ubuntu --version 24.04 --instance-type r1.xlarge --disk 100 --name gitea-vm --network 709a523d
+
+## wait for the cluster to be ready
+replicated vm ls
+---
+06db1e55    gitea-vm                       ubuntu          24.04         running         2025-02-18 17:29              2025-02-18 18:30        
+---
+## ssh into the VM with vm id
+ssh 06db1e55@replicatedvm.com
+```
+
+In the VM, you can verify the network connection to the PostgresSQL before deploying Gitea Helm chart.
+First, find the hostname of the PostgresSQL cluster.
+```bash
+replicated cluster ls
+---
+64ccba9c    postgres-cluster                embedded-cluster    5             running         2025-02-18 17:06              2025-02-18 21:12  
+---
+
+## connect to the kubeconfig of the PostgresSQL cluster
+replicated cluster shell 64ccba9c
+
+## get the hostname of the PostgresSQL cluster
+kubectl get node
+---
+64ccba9c1   Ready    control-plane   28m   v1.30.9+k0s
+---
+```
+
+`64ccba9c1` is the hostname of the PostgresSQL cluster. Use the hostname to verify the network connection to the PostgresSQL cluster:
+
+```bash
+sudo apt-get install -y postgresql-client
+## password is postgres
+psql -h 64ccba9c1 -p 5432 -U postgres -d postgres
+```
+
+After the network connection is verified, you can deploy Gitea Helm chart.
+
+Go to `applications/gitea` directory and run the following command to create a new release:
+```bash
+cd applications/gitea
+make release
+```
+
+Promote the release to the desired channel with a dedicated test user. The test user should have access to the Embedded Cluster.
```

Comment:
may be add details on using other customer mainly for gitea, in addition to existing customer from storagebox

Reply:
Good catch. I have added
```
7. Dedicated test channels named Storagebox and Gitea
8. Dedicated test customers for Storagebox and Gitea
```
In Prerequisites

And point out
Promote the release to the `Gitea` channel with `a dedicated test user` which is created recently.